common/src/main/java/com/microsoft/identity/common/exception/BaseException.java

```diff
@@ -150,6 +154,14 @@ public void setCorrelationId(@Nullable final String correlationId){
         mCorrelationId = correlationId;
     }
 
+    public String getUsername() {
+        return mUsername;
+    }
+
+    public void setUsername(String username) {
+        this.mUsername = username;
+    }
```

Comment:
Do we want to @Builderize this class or add a constructor with these additional arguments?

Reply:
No to both.

- What we're doing here is to inject username into an existing subclassed object. Using a builder here is sort of overkill IMO.
- This field is not common - it will set only in the interrupt flow the in-line WPJ scenario. Adding a new constructor here (and all the subclasses) would make the code unnecessarily cluttered. 

That said, I'll add Nullable on top of getUsername()